src/main/java/org/opentestsystem/shared/search/persistence/SearchableRepositoryImpl.java

```diff
@@ -14,15 +14,19 @@
 
 import org.opentestsystem.shared.search.domain.AbstractSearchRequest;
 import org.opentestsystem.shared.search.domain.SearchResponse;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.mongodb.core.MongoOperations;
 import org.springframework.data.mongodb.core.query.Query;
 import org.springframework.data.mongodb.repository.query.MongoEntityInformation;
 import org.springframework.data.mongodb.repository.support.SimpleMongoRepository;
+import org.springframework.stereotype.Component;
 
+@Component
 public class SearchableRepositoryImpl<T, ID extends Serializable> extends SimpleMongoRepository<T, ID> implements SearchableRepository<T, ID> {
     private final MongoEntityInformation<T, ID> entityInformation;
     private final MongoOperations mongoOperations;
 
+    @Autowired
```

Comment:
Same comment as above.  This was working without annotations so I'm curious why we're adding them now.  Or did we remove the XML config?